tasks/configure.yml

```diff
@@ -16,7 +16,7 @@
     group: "{{ postgresql_group }}"
     mode: 0600
   notify: restart postgresql
-  when: postgresql_hba_entries | bool
+  when: postgresql_hba_entries | length > 0
```

Comment:
+1 on not using bool
To catch all cases, I'ld add the mighty default filter in the chain:
```
   when: postgresql_hba_entries | default([]) | length > 0 
```

Reply:
I don't see the benefit. There it's already a default (non empty list) in `default/main.yml`.
If you overwrite that by sth not being a list you are screwed anyway.

Reply:
Agreed with @mdellweg - default is already `[]` so no need for belt + suspenders in this case.